org.tabos.roger.json

```diff
@@ -229,6 +233,12 @@
                     "url" : "https://gitlab.com/tabos/spandsp.git",
                     "tag" : "0.0.6pre21",
                     "commit" : "68455c2d0d56def0443ba2ae7df0e82e3a5fa5bb"
+                },
+                {
```

Comment:
I do not understand why this part is necessary, could you give more hints please?

Reply:
the `config.guess` and `config.sub` files are too old to detect `aarch64` and the build fail.

The thing that's weird here is that there are in git while usually these files are only disted in the tarball...